src/malcolm/reducer/malcolmReducer.test.js

```diff
@@ -346,4 +484,17 @@ describe('malcolm reducer', () => {
     );
     expect(attribute.errorState).toEqual(123);
   });
+
+  /* TODO: this needs to be fixed in the malcolm reducer
```

Comment:
Is this TODO to be done as part of this PR or a later one?

Reply:
as part of a later one (presumably the reducer refactor)